static/app/views/settings/project/projectReplays.tsx

```diff
@@ -80,6 +81,9 @@ function ProjectReplaySettings({organization, project, params: {projectId}}: Pro
         apiMethod="PUT"
         apiEndpoint={`/projects/${organization.slug}/${projectId}/`}
         initialData={project.options}
+        onSubmitSuccess={(
+          response // This will update our project context
+        ) => ProjectsStore.onUpdateSuccess(response)}
```

Comment:
Does this work?

```suggestion
        onSubmitSuccess={ProjectsStore.onUpdateSuccess}
```

Reply:
won't :(

https://github.com/getsentry/sentry/blob/d81a027147e7f97891f4e9f921d766b9912aded1/static/app/stores/projectsStore.tsx#L101-L103

Reply:
why not? @ryan953

Reply:
it's not using an arrow function expression to bind the `this` keyword, but it uses `this` inside the function body. Therefore we must use dot-syntax to bind the value of `this`.